src/components/AdvanceAnalyticsV2.0/AnalyticsFilters.jsx

```diff
@@ -67,10 +89,34 @@ const AnalyticsFilters = ({
                 <Form.Control
                   controlClassName="font-weight-normal analytics-filter-form-controls rounded-0"
                   as="select"
-                  disabled
-                  defaultValue=""
+                  onChange={(e) => handleDateRangeChange(e.target.value)}
+                  disabled={isFetching}
+                  defaultValue={DATE_RANGE.LAST_90_DAYS}
                 >
-                  <option value="">Custom</option>
+                  <option value={DATE_RANGE.LAST_7_DAYS}>
+                    {intl.formatMessage({
+                      id: 'advance.analytics.date.range.filter.option.last.7.days',
+                      defaultMessage: 'Last 7 days',
+                    })}
+                  </option>
+                  <option value={DATE_RANGE.LAST_30_DAYS}>
+                    {intl.formatMessage({
+                      id: 'advance.analytics.date.range.filter.option.last.30.days',
+                      defaultMessage: 'Last 30 days',
+                    })}
+                  </option>
+                  <option value={DATE_RANGE.LAST_90_DAYS}>
+                    {intl.formatMessage({
+                      id: 'advance.analytics.date.range.filter.option.last.90.days',
+                      defaultMessage: 'Last 90 days',
+                    })}
+                  </option>
+                  <option value={DATE_RANGE.YEAR_TO_DATE}>
+                    {intl.formatMessage({
+                      id: 'advance.analytics.date.range.filter.option.year.to.date',
+                      defaultMessage: 'Year to date',
+                    })}
+                  </option>
```

Comment:
Could you please add `description` in each of the `intl.formatMessage` call to explain the `defaultMessage`? Please [see](https://github.com/openedx/frontend-app-admin-portal/blob/569e9b55d9c770361f687349321f2c4e58b64d1a/src/components/Admin/AdminSearchForm.jsx#L117-L120) for an existing example